minigame/pkmnquiz_blue.asm

```diff
@@ -390,23 +392,24 @@ ENDM
 	lang D, next "3. TAKKI"
 
 	; French
-	lang F, db   "Quel est le nom du"
-	lang F, next "PIAFABEC origi-"
-	lang F, next "naire de JADIELLE?"
+	lang F, db   "Quel est le surnom"
+	lang F, next "du PIAFABEC de"
```

Comment:
Maybe “à” would be a little clearer? (Though either way, you've already improved a lot on my proposal :+1:)